src/main/java/org/springframework/data/util/ReflectionUtils.java

```diff
@@ -189,13 +193,32 @@ public static Field findField(Class<?> type, DescribedFieldFilter filter, boolea
 	}
 
 	/**
+	 * Finds the field of the given name on the given type.
+	 * 
+	 * @param type must not be {@literal null}.
+	 * @param name must not be {@literal null} or empty.
+	 * @return
+	 * @throws IllegalArgumentException in case the field can't be found.
+	 */
+	public static Field findRequiredField(Class<?> type, String name) {
```

Comment:
The corresponding methods in Spring Framework are called `findField(…)` which is why I used this pattern.